File: QE-Selenum java/demo/src/main/java/demos/Activity5.java
package demos;

public class 
Activity5 {
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;


    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));

        driver.get("https://training-support.net/webelements/dynamic-controls");
        System.out.println("Title: " + driver.getTitle());

        System.out.println("Checkbox displayed (before): " +
                driver.findElement(By.xpath("//*[@id='checkbox']")).isDisplayed());

        driver.findElement(
                By.xpath("/html/body/div/main/div/div/div/div/div[2]/section[1]/button")
        ).click();

        wait.until(ExpectedConditions.invisibilityOfElementLocated(
                By.xpath("//*[@id='checkbox']")));

        System.out.println("Checkbox displayed (after): False");

        driver.quit();
    }
}
}

File: QE-Selenium python/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()
wait = WebDriverWait(driver, 10)

driver.get("https://training-support.net/webelements/dynamic-controls")
print("Title:", driver.title)

print("Checkbox displayed (before):",
      driver.find_element(By.XPATH, "//*[@id='checkbox']").is_displayed())

driver.find_element(
    By.XPATH, "/html/body/div/main/div/div/div/div/div[2]/section[1]/button"
).click()

wait.until(EC.invisibility_of_element_located((By.XPATH, "//*[@id='checkbox']")))
print("Checkbox displayed (after): False")

driver.quit()